### DISCUSSION TITLE
Access connection from Cyberduck in Cryptomator

### DISCUSSION THREAD

**Discussion by HyperCriSiS** [iterate-ch/cyberduck#14927] [Q&A] [answered]
Hi,

why is it not possible in Cryptomator to access a connection made with Cyberduck?

Best regards
Hyper

**Comment by tatze96** [comment, marked as answer]
First of all, [Cryptomator](https://docs.cyberduck.io/cryptomator/) is fully integrated into Cyberduck and [Mountain Duck](https://mountainduck.io/), so you generally don't have to have the Cryptomator application installed separately.

Cyberduck uses its own file browser that isn't accessible from within other applications including the Cryptomator application. 
The vaults would be accessible from within the Cryptomator application while mounting them with Mountain Duck, given that the [automatic vault detection](https://docs.cyberduck.io/cryptomator/#auto-detect) is deactivated.
